### ISSUE TITLE
Can't even use the search function on your website.

### ISSUE THREAD

**Issue by nuaa520** [transitive-bullshit/nextjs-notion-starter-kit#539]
#### Description

I noticed that your project has a search button in the navigation bar. I wanted to test its functionality and found that it doesn't work. It keeps loading in a circle. I initially thought it was my own problem, but when I tested it on your website, I found that it also doesn't work.

#### Notion Test Page ID

I'm not actually doing my own project right now, just testing it in your website

![Shot 2023-08-31 at 17 36 41@2x](https://github.com/transitive-bullshit/nextjs-notion-starter-kit/assets/141616559/1b0b8661-6b54-4d29-9a45-4d2cece167b7)

**Comment by omarcostahamido** [comment]
same here. I just arrived to this project now (late to the party) and would love to be able to see this in action or figure out how to fix it... maybe worthwhile mentioning the previous issues as well.

**Comment by danni-cool** [comment]
I fixed the problem temporarily, and wrote an article, if you are interested, here's the link. https://danni.cool/fixed-the-broken-search-in-nextjs-notion-starter-kit

**Comment by danni-cool** [comment]
@omarcostahamido Thanks! I was struggling with the same problem too. I'm glad it could help.

**Comment by Le1a** [comment]
> 我暂时解决了这个问题，并写了一篇文章，如果你有兴趣，这里是链接。https://danni.cool/fixed-the-broken-search-in-nextjs-notion-starter-kit

I built it using Vercel+GitHub. I put your patch into GitHub and tried running it again using Vercel, but it still didn't work. I checked the network and found that it was 405 Method Not Authorized
![image](https://github.com/transitive-bullshit/nextjs-notion-starter-kit/assets/97610822/9e40252c-73a3-4c21-8277-0cb8824c6d7c)
![image](https://github.com/transitive-bullshit/nextjs-notion-starter-kit/assets/97610822/6e0e91e3-8155-4941-9236-f38844df7183)

**Comment by danni-cool** [comment]
@Le1a Have you ever installed `patch-packages` using NPM and added the pre-install hooks?

**Comment by Le1a** [comment]
> @Le1a Have you ever installed `patch-packages` using NPM and added the pre-install hooks?

No, because I don't know how to use the npm command in the GitHub project deployed in Vercel

**Comment by danni-cool** [comment]
@Le1a So you already know how to solve the problem.

**Comment by Le1a** [comment]
> @Le1a So you already know how to solve the problem.

How to solve this problem in Vercel

**Comment by Le1a** [comment]
> @Le1a Have you ever installed `patch-packages` using NPM and added the pre-install hooks?

It has been resolved. I found in Vercel that the install command can be entered during project construction. Enter npm i patch package -- legacy peer deps&&npm i -- legacy peer deps
![image](https://github.com/transitive-bullshit/nextjs-notion-starter-kit/assets/97610822/829bdfa1-8579-4506-9545-67d5dabe5644)

**Comment by danni-cool** [comment]
@Le1a Congratulations! you made it!

**Comment by chandragie** [comment]
> > @Le1a Have you ever installed `patch-packages` using NPM and added the pre-install hooks?
> 
> It has been resolved. I found in Vercel that the install command can be entered during project construction. Enter npm i patch package -- legacy peer deps&&npm i -- legacy peer deps
> ![image](https://github.com/transitive-bullshit/nextjs-notion-starter-kit/assets/97610822/829bdfa1-8579-4506-9545-67d5dabe5644)
> 

Hey, did you still declare the patch-package inside the package.json? My local using @danni-cool 's solution works, however deploying to Vercel is out of luck. Tried to add install command as you mentioned, still error.
